lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -9,52 +9,14 @@ function canMultiplyLikeTermConstantNodes(node) {
     return false;
   }
   const args = node.args;
-  if (!args.every(n => isConstantOrConstantPower(n))) {
+  if (!args.every(n => ConstantOrPowerTerm.isConstantOrConstantPower(n))) {
     return false;
   }
-  const constantTermList = node.map(n => getBaseNode(n));
-  const firstTerm = constantTermList.args[0];
-  const restTerms = constantTermList.args.slice(1);
+  const constantTermList = node.args.map(n => ConstantOrPowerTerm.getBaseNode(n));
```

Comment:
No it does the same thing, dont know why I wrote it the first way haha. Instead of looking at the args for the map I just checked them after, so it does the same thing now but much nicer.